app/controllers/spree/api/products_controller_decorator.rb

```diff
@@ -20,7 +20,7 @@ def bulk_products
 
     @products = product_query.order('created_at DESC').
       ransack(params[:q]).result.
-      page(params[:page]).per(params[:per_page])
+      page(params[:page] || 1).per(params[:per_page] || 50)
```

Comment:
magic numbers should be a constants at the top:
DEFAULT_PAGE = 1
DEFAULT_PER_PAGE = 50